repr.go

```diff
@@ -105,6 +105,12 @@ func Hide[T any]() Option {
 	}
 }
 
+func HideName(name string) Option {
```

Comment:
I don't mind the idea, but can you rename this to `HideField` or something similar, as "name" is a bit ambiguous - it could be field name or type name.

Reply:
No problem, ive done this now :)